src/components/BadTextInput.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <v-text-field
+    @change="change"
+    :disabled="disabled"
+    :label="label"
+    :placeholder="placeholder"
+    :id="id"
+    :value="value"
+  >
+  </v-text-field>
+</template>
+<script>
+export default {
+  name: "BadTextInput",
+  data() {
+    return {};
+  },
+  props: {
+    change: {
+      type: Function,
+      required: true
+    },
+    value: {
+      type: String,
+      default: ""
```

Comment:
When is of type String, you don't have to set the default value

Reply:
updated